library/psa_crypto_mac.c

```diff
@@ -468,11 +468,34 @@ static psa_status_t mac_update(
     const uint8_t *input,
     size_t input_length )
 {
-    /* To be fleshed out in a subsequent commit */
-    (void) operation;
-    (void) input;
-    (void) input_length;
-    return( PSA_ERROR_NOT_SUPPORTED );
+    if( ! operation->key_set )
+        return( PSA_ERROR_BAD_STATE );
```

Comment:
Actually, what this variable is guarding is already covered by alg != 0 for the mbedtls operation context, and id != 0 for the overarching PSA context. So I'll just remove the `key_set` and associated accesses, since it's superfluous.